Gemfile

```diff
@@ -0,0 +1,24 @@
+source "https://rubygems.org"
+
+group :test do
+  gem "rake"
+  gem "puppet", :git => 'https://github.com/puppetlabs/puppet.git', :tag => '3.8.0'
```

Comment:
Yes, this is broken. There are no Puppet 3.8 gem releases yet, so this wouldn't work in a travis environment anyway.